src/components/VaultsPanel.tsx

```diff
@@ -69,11 +71,25 @@ export default function VaultsPanel() {
         return <BurnIst charterOfferId={charterOfferId} />;
       case ProposalTypes.pauseLiquidations:
         return <PauseLiquidations charterOfferId={charterOfferId} />;
+      case ProposalTypes.auctioneerParamChange:
+        return <AuctioneerParamChange charterOfferId={charterOfferId} />;
       default:
         return <div>TODO</div>;
     }
   })();
 
+  // Don't allow certain proposal types on mainnet.
+  const proposalTypesForSelectedNetwork = (() => {
+    const values = Object.values(ProposalTypes);
+    if (walletUtils.agoricNet !== 'main') {
+      return values;
+    }
+
+    return Object.values(ProposalTypes).filter(
+      v => v !== ProposalTypes.auctioneerParamChange,
+    );
+  })();
```

Comment:
In both cases it's iterating over values. It's basically a different filter. The filter is a policy so it should be up at module scope or a config module., E.g.,
```js
const networkProposalFilter = () => {
  if (walletUtils.agoricNet === 'main') {
    return p => p !== ProposalTypes.auctioneerParamChange;
  }
  // no filter
  return Boolean;
}
```

Reply:
Done